iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/fragment/FragmentInstanceExecution.java

```diff
@@ -57,6 +70,8 @@ public class FragmentInstanceExecution {
   private final long timeoutInMs;
 
   private final MPPDataExchangeManager exchangeManager;
+  private final ReadWriteLock statisticsLock = new ReentrantReadWriteLock();
```

Comment:
we don't need this lock, we only need to change `staticsRemoved` to `AtomicBoolean`.